internal/saucecloud/espresso.go

```diff
@@ -207,9 +207,10 @@ func (r *EspressoRunner) startJob(jobOpts chan<- job.StartOptions, s espresso.Su
 		DeviceType:        d.deviceType,
 		DevicePrivateOnly: d.privateOnly,
 
-		// Instrumentation options, for RDC only
-		AudioCapture:   r.Project.Instrumentation.AudioCapture,
-		NetworkCapture: r.Project.Instrumentation.NetworkCapture,
+		// Overwrite device settings
+		RealDeviceKind: strings.ToLower(espresso.Android),
+		AudioCapture:   s.AppSettings.AudioCapture,
+		NetworkCapture: s.AppSettings.Instrumentation.NetworkCapture,
```

Comment:
Isn't it better to send over the entire `AppSettings` struct? I don't see the need to flatten out the structure here, considering that internally, these settings are combined again into app settings 🤔

Reply:
ok. I'll update it.